package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "task_manager",
+  "version": "1.0.0",
+  "description": "This is simple task manager that you can create to-do lists for projects.",
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "webpack serve --open",
```

Comment:
`"build": "webpack --mode production",` this would help to run the `npm run build command` for deployment